Duplicate names attachments Browser.tpl

loma-one opened the pull request:
Problem: The browser view for attachments displayed the attachment name twice. With the change, only the filename is shown, which improves clarity.

Browser.php

fix: correct file type detection and extend file array in media attachment browser

- Fixed incorrect use of `file_exists()` with `sprintf()` in a ternary expression.
  Replaced with more readable and valid logic using string concatenation.
- Extended the return value of `map_files()` by adding a fourth element (filename),
  likely for enhanced display or sorting functionality in the frontend.
- Introduced intermediate variables `$iconUrl` and `$fileUrl` to improve code readability
  and avoid duplication.

These changes improve code robustness, readability, and prepare the module for potential UI enhancements.

browser.tpl

Improve image display in file browser

- Removed image caption and added filename as title attribute in <img> tag
- Changed alt attribute to use image description instead of filename

loma-one committed:
Duplicate names attachments

Improve image display in file browser

- Removed image caption and added filename as title attribute in <img> tag
- Changed alt attribute to use image description instead of filename

annando commented on view/theme/frio/templates/media/browser.tpl:
```diff
@@ -53,14 +53,14 @@
 			</div>
-		</div>
+			</div>
+			</div>
```
Had one `</div>` been missing?

loma-one replied:
Oh yes, there were two superfluous and incorrectly placed tags. I have removed them

loma-one committed:
corrects div

corrects the incorrect nesting of tags in the HTML code by removing two superfluous and incorrectly placed closing tags

marcusxss commented on view/theme/frio/templates/media/browser.tpl:
```diff
@@ -53,15 +53,13 @@
-						</a>
-					</div>
+							<img src="{{$f.2}}" alt="{{$f.3}}" title="{{$f.1}}">
+							</a>
+						</div>
```
These should be left-indented one level to match the opening tags.

marcusxss commented:
A bit curious why this PR lists changes to Browser.php when they're split into two PRs?

tobiasd committed:
bumb RC version

tobiasd committed:
extend valitity or security.txt

tobiasd committed:
bump version

marcusxss committed:
BBCode: Remove duplicate class attribute

annando committed:
Merge pull request #14994 from mf-fx/remove_duplicate_class_attribute

BBCode: Remove duplicate class attribute

loma-one committed:
Merge branch 'friendica:2025.07-rc' into loma-one-patch-2
